anchor-service/src/anchor.rs

```diff
@@ -1,16 +1,36 @@
+use anyhow::{anyhow, Context, Result};
 use cid::Cid;
 use indexmap::IndexMap;
-use serde::{Deserialize, Serialize};
+use ipld_core::ipld::Ipld;
+use serde::Serialize;
+use std::fmt::Debug;
+use tracing::info;
 
-use ceramic_event::unvalidated::{Proof, RawTimeEvent};
+use ceramic_core::{EventId, SerializeExt};
+use ceramic_event::unvalidated::{Proof, ProofEdge, RawTimeEvent, TimeEvent};
 
 /// AnchorRequest for a Data Event on a Stream
-#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[derive(Clone, PartialEq, Eq, Serialize)]
 pub struct AnchorRequest {
-    /// The CID of the stream
+    /// The CID of the Init Event of the stream
     pub id: Cid,
     /// The CID of the Event to be anchored
     pub prev: Cid,
+    /// ID for the Event being anchored
+    pub event_id: EventId,
+    /// the ResumeToken for the Event being anchored in the local database.
+    pub resume_token: i64,
+}
+
+impl Debug for AnchorRequest {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("AnchorRequest")
+            .field("id", &format!("{:?}", &self.id))
+            .field("prev", &format!("{:?}", &self.prev))
+            .field("order_key", &format!("{:?}", &self.event_id))
```

Comment:
why not rename of the event_id field to order_key ?

Reply:
we still talk about `ceramic_core::EventId` in some places as an "order keys" and in some places as "event ids". We should pick one. ... my preference to use order_key everywhere.

Reply:
Works for me, I say rename the field now. Then in other places can be renamed as we go along. Gotta start somewhere.